### ISSUE TITLE
New filter blocks should fallback to filtering all products of the store

### ISSUE THREAD

**Issue by dinhtungdu** [woothemes/woocommerce#44753]
When added to non-product archive templates and normal pages or outside of the product collection block, new filter blocks should still work and we can use the all products query as the filter target.

As @nerrad suggested:

> Also, I think it’d be fine to have the filter blocks affect the query for all product collection blocks on the page if not inside the Product Collection tree. Guidance can be provided in the Inspector Control

**Comment by dinhtungdu** [comment]
If we decide to do this, we need to communicate clearly with merchants. As the query is possibly mismatched now: the Product Collection can have its own query, but new filter blocks (when added outside the Product Collection block) will fall back to all products.

**Comment by nerrad** [comment]
> If we decide to do this, we need to communicate clearly with merchants

Let's make sure we get some UX feedback from @jarekmorawski on the placement for this type of communication (and we might need to get some copy review as well).

**Comment by jarekmorawski** [comment]
I believe this issue will be solved by https://github.com/woocommerce/woocommerce/issues/42073. I'd start by not letting merchants add the filters block in a non-archive template unless they nest it inside Product Collection. While admittedly limiting, it'd simplify the setup flows and strengthen the connection between the filters and the Product Collection block, which they cannot function without.

**Comment by nerrad** [comment]
>  I'd start by not letting merchants add the filters block in a non-archive template unless they nest it inside Product Collection.

I'd simplify further and restrict them only to archive templates (where they can be used anywhere in the template because the context is known). This is the typical use for shopper-facing filters anyway.

**Comment by jarekmorawski** [comment]
I agree. This could be a good starting point and it's what I'm proposing in the new designs. We can reiterate based on data and user feedback.

**Comment by kkoteen** [comment]
@dinhtungdu could you take a look as the original reporter and suggest next steps? Is it still needed? Thanks!

**Comment by dinhtungdu** [comment]
This can be closed now as implemented @kkoteen
